packages/orchestration/src/exos/chain-hub.js

```diff
@@ -216,6 +217,11 @@ export const makeChainHub = (zone, agoricNames, vowTools) => {
     keyShape: BrandShape,
     valueShape: M.string(),
   });
+  /** @type {MapStore<string, string>} */
+  const bech32PrefixToChainName = zone.mapStore('bech32PrefixToChainName', {
```

Comment:
The 1st arg to `makeScalarMapStore` is a tag for just the key, not the whole map, for use in error messages such as `bech32Prefix not found: blort`. So I was going to suggest:

```suggestion
  const bech32PrefixToChainName = zone.mapStore('bech32Prefix', {
```

but it looks like the arg here is a baggage key, so the longer name seems right.